lua/lspconfig/haxe_language_server.lua

```diff
@@ -0,0 +1,61 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.haxe_language_server = {
+  default_config = {
+    cmd = {"haxe-language-server"};
+    filetypes = {"haxe"};
+    root_dir = util.root_pattern("build.hxml");
+    settings = {
+      ["haxe.executable"] = "haxe";
+    };
+    init_options = {
+      displayArguments = {"build.hxml"};
+    };
+  };
+  docs = {
+    description = [[
+https://github.com/vshaxe/haxe-language-server
+
+Though bundled with the Visual Studio Code Haxe Extension, the Haxe
+language server can be built as a standalone project by cloning the
+repo and then running the following commands from the project's root
+directory:
+
+    npm install
+    npx lix run vshaxe-build -t language-server
+
+This will create a JavaScript file at `bin/server.js` that can be
```

Comment:
This will create `bin/server.js` sounds less verbose, idk you call on this one

Reply:
Sounds good to me.